pkgs/applications/misc/tuckr/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tuckr";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "RaphGL";
+    repo = pname;
+    rev = "${version}";
```

```suggestion
    repo = "Tuckr";
    rev = version;
```

It is not useful to parameterize `pname`. And this is a clear example why: the repo name is different from pname. (And no, you should not rely on "auto resolution" of URLs).

Also, `"${attr}"` is just the same as `attr`.